CONTRIBUTING.md

```diff
@@ -35,9 +36,32 @@ times in N different commits. If there was some accidental reformatting or white
 changes during the course of your commits, please rebase them away before submitting
 the PR.
 
+### CHANGELOG using Beachball
+
+[Beachball](https://microsoft.github.io/beachball/) is a semantic version bumper that also has an automated tool to ask devs to log changes in a simple CLI manner.
```

Comment:
Nit: can you use "contributors" instead of "devs" please?

Reply:
Done, will be included in the next commit(s)!

Reply:
Updated